conf/defaults.ini

```diff
@@ -229,6 +229,8 @@ snapshot_remove_expired = true
 [dashboards]
 # Number dashboard versions to keep (per dashboard). Default: 20, Minimum: 1
 versions_to_keep = 20
+# Minimum dashboard refresh rate
+min_refresh_rate_duration = 5m
```

Comment:
min_refresh_interval   seems like a better naming option as we usually refer to this as an interval in the frontend

Reply:
Hello @torkelo, among the last changes was this naming update, which has been discussed here: https://github.com/grafana/grafana/pull/19416#discussion_r357277392 
Could you eventually discuss this with @marefr there ?

Reply:
I still vote for having duration at the end to denote and help you understand this settings takes duration as parameter, i.e. 5s or 1d. But maybe `min_refresh_interval_duration` then? If @torkelo disagrees with me I vote for `min_refresh_interval` and trying to get this merged.

Reply:
we do not suffix any other interval setting names with duration (in ds settings, query editor UI, playlist, dashboard settings). not sure duration makes it clearer what the value is, still needs examples/description to make formating clear, but maybe durationo name makes it a little clearer? But worried about inconsistent naming as interval is the name we currently use for time periods.

Reply:
I think I've mixed this up with another PR where we're adding the duration-suffix. As I wrote fine by me to name it `min_refresh_interval`.

Reply:
I renamed all of the occurences of this term to minRefreshInterval.
thanks for having synced so fast !